colorschemes/octagon.conf

```diff
@@ -0,0 +1,117 @@
+# Octagon is a theme inspired by https://github.com/loctvl842/monokai-pro.nvim:
+# Colors selected by DrewG @ justaguylinux(dot)com
+
+
+[theme_info]
+name=Octagon
+description=A dark theme using Monokai Pro Octagon color palette
+# incremented automatically, do not change manually
+version=1
+
+[named_colors]
+bg=#282a3a
+bg+1=#3a3d4b
+bg+2=#535763
+fg-4=#696d77
+fg-3=#888d94
+fg-2=#a0a5ae
+fg-1=#eaf2f1
+fg=#f5f9f8
+
+white=#ffffff
+red=#ff657a
+orange=#ff9b5e
+yellow=#ffd76d
+green=#bad761
+blue=#9cd1bb
+purple=#c39ac9
+pink=#ff657a
+
+[named_styles]
+default=fg;bg;false;false
+error=red;bg+1;false;false
+
+# Editor styles
+#-------------------------------------------------------------------------------
+
+selection=bg+2;fg-4;false;true
+current_line=bg+1;;true;false
+brace_good=blue;;true;false
+brace_bad=red;bg;true;false
+margin_line_number=fg-3;bg+1;false;false
+margin_folding=fg-4;bg+2;false;false
+fold_symbol_highlight=fg-3
+indent_guide=fg-4
+caret=fg;bg;false
+marker_line=fg-2;yellow
+marker_search=blue;bg+1;false;false
+marker_mark=orange;bg;false;false
+call_tips=fg-1;bg+2;false;false
+white_space=fg-4;;true
+
+# Programming languages
+#-------------------------------------------------------------------------------
+
+comment=fg-3;;;true
+comment_doc=fg-3;bg+1;false;false
+comment_line=comment
+comment_line_doc=comment_doc
+comment_doc_keyword=comment_doc,bold
+comment_doc_keyword_error=comment_doc,italic
+
+number=green
+number_1=number
+number_2=number_1
+
+type=purple;;true;false
+class=type
+function=type
+parameter=function
+
+keyword=orange;;true;false
+keyword_1=keyword
+keyword_2=yellow;;true;false
+keyword_3=keyword_1
+keyword_4=keyword_1
+
+identifier=default
+identifier_1=identifier
+identifier_2=identifier_1
+identifier_3=identifier_1
+identifier_4=identifier_1
+
+string=red
+string_1=string
+string_2=string_1
+string_3=string_1
+string_4=string_1
+string_eol=fg;background;false;false
```

Comment:
Geany comlains here that `background` is not valid:

`Geany-INFO: 17:26:09.342: Bad color 'background'`